### DISCUSSION TITLE
CSS Isolation with Havit Controls?

### DISCUSSION THREAD

**Discussion by chobo2** [havit/Havit.Blazor#671] [Q&A] [answered]
Hi

I am using Blazor 8 + interactiveServer but noticed my css is not coming through to my controls.

```
  <h1 class="test">TEST</h1>
   <HxSubmit Text="Submit" Color="ThemeColor.Primary" OnValidClick="async () => await Task.Delay(1000)"  CssClass="test"/>

<style>
  .test {
    font-size: 500px;
  }
</style
```

This works when it is in the same razor page. If say this page is called home.razor, and I create a page called home.razor.css (ie doing isolation of css).

Only the h1 tag will get the style applied to it.

**Comment by crdo** [comment, marked as answer]
You have to use ::deep as per Blazor docs. https://learn.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-7.0

**Reply by chobo2** [reply]
cool, will check it out.
